src/main/java/Main.java

```diff
@@ -0,0 +1,30 @@
+//@Override
+
+import java.io.IOException;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke("data/duke.txt");
```

Comment:
It is better to define the file name in a single place as constant if it is not going to be changed